arrows/gdal/image_container.cxx

```diff
@@ -240,6 +240,30 @@ ::get_image() const
   return img;
 }
 
+// ----------------------------------------------------------------------------
+/// Return a cropped vital image. Unlike other image container must allocate memory.
```

Comment:
minor nitpick: This summary comment does not match the one in the header, so I think Doxygen will generate a description that concatenates both comments together, which is probably no what we want.